css-contain-2/Overview.bs

```diff
@@ -1970,9 +1989,13 @@ Restrictions and Clarifications {#cv-notes}
 		and not cause any forced layouts.
 	</div>
 
-4. The initial determination of visibility for ''content-visibility: auto'' must
-    happen in the same frame that determined an existence of a new
-    ''content-visibility: auto'' element.
+4. Elements with ''content-visibility: auto''
+	that have not determined [=proximity to the viewport=]
+	must determine their proximity to the viewport
+	in the next <a href="https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering">Update the Rendering</a> cycle. 
+	The effect of this determination must be reflected in the next visual update.
```

Comment:
This *seems* to be saying something different from what the old spec text was saying, although I'm not sure.  (It seems like it's saying it's deferred one update cycle now, whereas it was immediate before.)  Is a difference intended?

Reply:
"next" is ambiguous here I think. I've changed the wording a bit. The idea is that the element could be created at any point in time, the very next Update the Rendering should determine its proximity. The visual update that results from this "update the rendering" should include the effects of the proximity change to relevance to the user.

I'll update the text